functions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+    /**
+    * Creates a connection to the collectorapp database
```

Comment:
Missing return type in doc block - if you just regenerate the docblock phpstorm will add it for you